src/FunderMaps.Data/Repositories/BundleRepository.cs

```diff
@@ -0,0 +1,304 @@
+        ///     Creates a new <see cref="Bundle"/> in our database.
+        /// </summary>
+        /// <param name="entity">The bundle to create.</param>
+        /// <returns>The id of the created bundle.</returns>
+        public override async ValueTask<Guid> AddAsync(Bundle entity)
+        {
+            if (entity == null)
+            {
+                throw new ArgumentNullException(nameof(entity));
+            }
+
+            var sql = @"
+                INSERT INTO maplayer.bundle(
+                    organization_id,
+                    name,
+                    user_id,
+                    layers)
+                VALUES (
+                    @organization_id,
+                    @name,
+                    @user_id,
+                    @layers)
+                RETURNING id";
+
+            await using var context = await DbContextFactory(sql);
+
+            MapToWriter(context, entity);
+
+            await using var reader = await context.ReaderAsync();
+
+            return reader.GetGuid(0);
+        }
+
+        /// <summary>
+        ///     Retrieve number of entities.
+        /// </summary>
+        /// <returns>Number of entities.</returns>
+        public override async ValueTask<long> CountAsync()
+        {
+            var sql = @"
+                SELECT  COUNT(*)
+                FROM    maplayer.bundle";
+
+            await using var context = await DbContextFactory(sql);
+
+            return await context.ScalarAsync<long>();
+        }
+
+        /// <summary>
+        ///     Delete <see cref="Bundle"/>.
+        /// </summary>
+        /// <param name="entity">Entity object.</param>
+        public override async ValueTask DeleteAsync(Guid id)
+        {
+            ResetCacheEntity(id);
+
+            var sql = @"
+                DELETE
+                FROM    maplayer.bundle
+                WHERE   id = @id";
+
+            await using var context = await DbContextFactory(sql);
+
+            context.AddParameterWithValue("id", id);
+
+            await context.NonQueryAsync();
+        }
+
+        /// <summary>
+        ///     Gets a <see cref="Bundle"/> by its id.
+        /// </summary>
+        /// <param name="id">Internal bundle id.</param>
+        /// <returns>The retrieved bundle.</returns>
+        public override async ValueTask<Bundle> GetByIdAsync(Guid id)
+        {
+            if (id == null || id == Guid.Empty)
+            {
+                throw new ArgumentNullException(nameof(id));
+            }
+
+            var sql = @"
+                SELECT  id,
+                        organization_id,
+                        name,
+                        create_date,
+                        update_date,
+                        delete_date,
+                        user_id,
+                        layer_configuration,
+                        version_id,
+                        bundle_status
+                FROM    maplayer.bundle
+                WHERE   id = @id";
```

Comment:
```suggestion
            var sql = @"
                SELECT  *
                FROM    maplayer.bundle
                WHERE   id = @id";
```

Reply:
The query already requests every single column that exists in the bundle table. I can see why explicitly declaring them would be more futureproof (in case a column is added that we don't want to request), however I until then I would just use an asterisk.